### DISCUSSION TITLE
Create types for `telnetlib`

### DISCUSSION THREAD

**Discussion by ghost** [DefinitelyTyped/DefinitelyTyped#57417] [Request a new @types package]
`telnetlib` is a telnet library for Node.JS, and I need it for my project, here is the NPM page:
https://www.npmjs.com/package/telnetlib
Here is the GH page:
https://github.com/cadpnq/telnetlib

**Comment by martin-badin** [comment]
HI @pjalsGit. I created PR based on your request. #57422
